proto/minder/v1/minder.proto

```diff
@@ -1000,6 +1000,14 @@ message RuleEvaluationStatus {
     string rule_type_name = 13;
     // rule_description_name is the name to describe the rule
     string rule_description_name = 14;
+    // alert_status is the status of the alert
+    string alert_status = 15;
+    // alert_last_updated is the last time the alert was performed or attempted
+    optional google.protobuf.Timestamp alert_last_updated = 16;
+    // alert_details is the description of the alert attempt if any
+    string alert_details = 17;
+    // alert_metadata is the metadata of the alert
+    map<string, string> alert_metadata = 18;
```

Comment:
This is the same content that is in the GitHub Security Alert?

Do you want to put all this in its own message?

Reply:
This is going to be `"ghsa_id":"GHSA-m2fw-9hxg-hm6p"` in the case of GHSA alert types, otherwise it can be something else for other alert types. 

My reasoning was if we decide to show a hyperlink to the alert we created from the UI.

Reply:
Is this too much or it doesn't make sense to share that yet?

Reply:
I think including the ID of the security alert (or even the URL) makes sense.  I worry about `map<string, string> metadata` constructs because they remove the ability to actually understand the contents of the message from the schema, so we end up needing to manually work around the fact that we don't get auto-generated schema documentation.

If this were a message instead, you would write:

```proto
message AlertMetadata {
  string ghsa_id = 1;
}

...

  AlertMetadata metadata = 18;
}
```

And it would serialize the same way for JSON, but it would be clear what metadata fields were present in the schema.

Reply:
(Since you haven't pushed yet, I was also suggesting we might want to do:

```
message EvalResultAlert {
    // status is the status of the alert
    string status = 1;
    // alert_last_updated is the last time the alert was performed or attempted
    optional google.protobuf.Timestamp last_updated = 2;
    // alert_details is the description of the alert attempt if any
    string details = 3;
    // alert_url is a URL pointing to the alert, in the context of the system it was filed in.
    string alert_url = 4;
    // OR
    // id is the id of the alert in an upstream system, such as "GHSA-m2fw-9hxg-hm6p" for GitHub
    string id = 5; 
}

message RuleEvaluationStatus {
  ...

  // If the rule generated an alert in an external system, alert will include the details of that alert.
  EvalResultAlert alert = 15;
}

```

)